react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText
+}) => (
+  <button className="ma__button-alert" onClick={onClick}>
+    <span className="ma__button-alert__show">{showText}</span>
+    <span className="ma__button-alert__hide">{hideText}</span>
+    <Icon />
+    {text}
+  </button>
+);
+
+ButtonAlert.propTypes = {
+  text: PropTypes.string,
+  onClick: PropTypes.func,
+  expanded: PropTypes.bool,
+  showText: PropTypes.string,
+  hideText: PropTypes.string
```

Comment:
This is also marked as required:
```suggestion
  hideText: PropTypes.string.isRequired
```